app_feup/lib/view/Pages/schedule_page_view.dart

```diff
@@ -113,4 +113,5 @@ class SchedulePageView extends StatelessWidget {
           Center(child: Text('Não possui aulas à ' + daysOfTheWeek[day] + '.')),
     );
   }
+
```

Comment:
Unnecessary empty space, can you remove it?

Reply:
done